gsplat/compression/compress.py

```diff
@@ -0,0 +1,340 @@
+import os
+from typing import Any
+import json
+import torch
+from torch import Tensor
+import torch.nn.functional as F
+from torchpq.clustering import KMeans
+import numpy as np
+import imageio
+
+from plas import sort_with_plas
```

Comment:
Add an try-except for this import and raise the error with an helper message on how to install it.

Reply:
Added